src/extension.ts

```diff
@@ -67,11 +68,16 @@ export function activate(context: ExtensionContext) {
 	window.onDidChangeActiveTextEditor((editor) =>{
 		toggleItem(editor, item);
 	});
-});
+
+	});
```

Comment:
it seems there is an issue with indentation previously. To not have modification on a lot of unrelated lines i cahnged only one which is mitigating the issue.